code/modules/hydroponics/seed_datums.dm

```diff
@@ -184,7 +184,7 @@
 	display_name = "killer tomato plant"
 	mutants = null
 	can_self_harvest = 1
-	has_mob_product = /mob/living/simple_animal/tomato
+	has_mob_product = /mob/living/simple_animal/hostile/tomato
```

Comment:
I should also clarify: I removed the hostile path from retaliate-only mobs. Having the path be "hostile/retaliate" is a bit unusual, as they ARE NOT hostile under normal circumstances and pathing them as such is strange. I mov